composer.json

```diff
@@ -28,29 +28,22 @@
     },
     "sort-packages": true
   },
-  "extra": {
-    "downloads": {
-      "phpstan": {
-        "url": "https://github.com/phpstan/phpstan/releases/latest/download/phpstan.phar",
-        "path": "vendor/bin/phpstan",
-        "type": "phar"
-      }
-    }
-  },
   "autoload": {
     "psr-4": {
       "Google\\Web_Stories\\": "includes"
     }
   },
   "autoload-dev": {
     "psr-4": {
-      "Google\\Web_Stories\\Tests\\": "tests/phpunit/includes",
-      "PHPStan\\WordPress\\": "tests/phpstan/src"
+      "Google\\Web_Stories\\Tests\\": "tests/phpunit/includes"
     }
   },
   "prefer-stable": true,
   "scripts": {
     "phpcs": "phpcs",
-    "phpstan": "phpstan analyse --memory-limit=512M"
+    "phpstan": [
+      "composer show --path szepeviktor/phpstan-wordpress || composer require --dev --ignore-platform-reqs szepeviktor/phpstan-wordpress",
```

Comment:
So now if I run `composer phpstan` it will modify the `composer.json` and `composer.lock` files? That's not really ideal.

Reply:
You got me!

Reply:
@swissspidy You got me so hard I had to eat 🥯 to keep going.

Reply:
Done.